configuration/parse.go

```diff
@@ -128,7 +173,17 @@ Options:
 		os.Exit(2)
 	}
 
-	err := flag.CommandLine.Parse(args)
+	usersHomeDir, _ := os.UserHomeDir()
+
+	path := filepath.Join(usersHomeDir, types.OssIndexDirName, types.OssIndexConfigFileName)
+
```

Comment:
I'm hesitant to allow this, because by using a common config location, the rest of the tools (`jake`, etc...) can pick up the settings and run with them. That makes it easy for someone to use all the tools, some of the tools, etc...

WDYT

Reply:
Makes sense. 👍